signingscript/Dockerfile

```diff
@@ -4,17 +4,19 @@ RUN groupadd --gid 10001 app && \
     useradd -g app --uid 10001 --shell /usr/sbin/nologin --create-home --home-dir /app app
 
 RUN apt-get update \
- && apt-get install -y zipalign osslsigncode \
+ && apt-get install -y zipalign osslsigncode cmake clang \
  && apt-get clean \
  && ln -s /app/docker.d/healthcheck /bin/healthcheck
 
 COPY . /app
 RUN chown -R app:app /app
 
+RUN cd /app/signingscript/docker.d && bash build_msix_packaging.sh && cp msix-packaging/.vs/bin/makemsix /usr/bin && cp msix-packaging/.vs/lib/libmsix.so /usr/lib && cd .. && rm -rf msix-packaging
```

Comment:
I don't love a) compile tools in the signingscript docker image, and b) multiple compiles of this tool, which may mean we get variance between docker image builds (unless we have reproducible builds here). If we're thinking this tool may stay static for a while, installing a pre-compiled tool may be preferable. If we're not sure, this is probably an acceptable short- to medium- term thing.

Reply:
Where would you prefer they be built? The main alternative I can think of is someone's laptop, which seems worse. We don't have any CI stuff for our `msix-packaging` fork.

Reply:
Hm, I suppose we could create a build-msix-packaging toolchain task that the k8s-image task downloads an artifact from. We don't have to block on this.